### DISCUSSION TITLE
Add battery percentage on G-Helper's main screen...

### DISCUSSION THREAD

**Discussion by Noisrevid** [seerge/g-helper#972] [Ideas]
Just a simple addition of adding the actual battery percentage on the G-Helper main screen..    I like that we can see discharge and charging rates but it doesn't tell me where my battery % is at..   Call me lazy, but I don't like having to either hover over the Windows battery icon and or open the Power Options menu to see my %.     Perhaps a simply actual Battery percentage next to the Discharging/Charging area would be nice.

@seerge Please?  :)

**Comment by seerge** [comment]
@Noisrevid i can add it like here, but i'm not sure about layout

[GHelper.zip](https://github.com/seerge/g-helper/files/12252328/GHelper.zip)
